static-analysis-plugin/plugin/src/test/groovy/com/novoda/test/LogsSubject.groovy

```diff
@@ -25,51 +27,56 @@ class LogsSubject extends Subject<LogsSubject, Logs> {
         Truth.assertAbout(FACTORY).that(logs)
     }
 
+    public static LogsSubject assumeThat(Logs logs) {
```

Comment:
I suppose this is a side-effect of being a Java programmer, where defaults are different. I don't mind to push for something more idiomatic, but I wonder if in this case it wouldn't make more sense to be explicit to identify public APIs.